### PR TITLE
Frontend/put card

### PR DESCRIPTION
**Pull request by monikasudol**

### CODE REVIEW

File: front/src/utils/request.js [hunk 1]
```diff
@@ -10,3 +10,11 @@ export const post = (url, body, additionalConfig = {}) => fetch(url, {
...
 
 export const toJSON = (response) => response.json();
+
+export const get = (url) => fetch(url, {
+  method: 'GET',
+  'credentials': 'include',
+  headers: {
+    'Content-Type': 'application/json'
+  }
+});
```
Comment (yuriyyakym) [inline front/src/utils/request.js]: Missing empty line

Comment (monikasudol) [reply front/src/utils/request.js]: done

File: front/src/utils/request.js [hunk 2]
```diff
@@ -11,10 +11,12 @@ export const post = (url, body, additionalConfig = {}) => fetch(url, {
...
   'credentials': 'include',
+  ...additionalConfig,
   headers: {
-    'Content-Type': 'application/json'
+    'Content-Type': 'application/json',
+    ...additionalConfig.headers
   }
-});
\ No newline at end of file
+});
```
Comment (yuriyyakym) [inline front/src/utils/request.js]: Very nice empty line

**Comment by yuriyyakym** [comment]
Just fix conflicts and we can merge it

File: front/src/utils/request.js (hunk 2 above)
Comment (monikasudol) [reply front/src/utils/request.js]: i have done everything i can to fix that

**Pull request merged**